WebApiDemo/WebApiDemo/Database/BookData.cs

```diff
@@ -0,0 +1,56 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+using WebApiDemo.Model;
+
+namespace WebApiDemo.Database
+{
+    public class BookData
+    {
+        public static List<Book> bookList = new List<Book>()
+        {
+            new Book(){ Name = "C# Basics", Author="PK", Category="CSE", Id = 101, Price=500},
+            new Book(){ Name = "Microprocessor", Author="Sinha", Category="Electronics", Id = 73, Price=792}
+        };
+
+        public List<Book> GetBooks()
+        {
+            return bookList;
+        }
+
+        public Book GetBookById(int id)
+        {
+            foreach (Book book in bookList)
```

Comment:
Better way to implement the same:
Book book=bookList.Find(b => b.Id == id);
return book;